src/it/java/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/GenericWhitespaceTest.java

```diff
@@ -54,6 +54,7 @@ public void testWhitespaceAroundGenerics() throws Exception {
             "14:46: " + getCheckMessage(messages, msgPreceded, ">"),
             "15:33: " + getCheckMessage(messages, msgFollowed, "<"),
             "15:33: " + getCheckMessage(messages, msgPreceded, "<"),
+            "15:46: " + getCheckMessage(messages, msgFollowed, ">"),
```

Comment:
All the similar additions in expected violations is due to following:
In this instance the extra violation is in the following line:
https://github.com/checkstyle/checkstyle/blob/9842608983cf00c1927d5e5bfcffdb05f4936afb/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAroundGenerics.java#L15
Earlier check didn't considered the whitespace between `>` `(` as a violation, this is now correctly identified in the fix.